File: Code/java/sign.java
import java.util.Scanner;

class Sign
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Number: ");
        float num = scanner.nextFloat();

        if (num < 0)
            System.out.println("Negative");
        else if (num > 0)
            System.out.println("Positive");
        else
            System.out.println("Zero");

        scanner.close();
    }
}

File: Code/cpp/sign.cpp
#include<iostream>
using namespace std;
int main()
{
    float num;

    cout << "Enter Number: ";
    cin >> num;

    if (num < 0)
        cout << "Negative";
    else if (num > 0)
        cout << "Positive";
    else
        cout << "Zero";
    return 0;
}